mmv1/products/iam3/FoldersPolicyBinding.yaml

```diff
@@ -0,0 +1,190 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: 'FoldersPolicyBinding'
+description: A policy binding to a folder
+references:
+  guides:
+    'Apply a policy binding': 'https://cloud.google.com/iam/docs/principal-access-boundary-policies-create#create_binding'
+  api: 'https://cloud.google.com/iam/docs/reference/rest/v3beta/folders.locations.policyBindings'
+min_version: 'beta'
+id_format: 'folders/{{folder}}/locations/{{location}}/policyBindings/{{policy_binding_id}}'
+base_url: 'folders/{{folder}}/locations/{{location}}/policyBindings'
+self_link: 'folders/{{folder}}/locations/{{location}}/policyBindings/{{policy_binding_id}}'
+create_url: 'folders/{{folder}}/locations/{{location}}/policyBindings?policyBindingId={{policy_binding_id}}'
+update_verb: 'PATCH'
+update_mask: true
+import_format:
+  - 'folders/{{folder}}/locations/{{location}}/policyBindings/{{policy_binding_id}}'
+timeouts:
+  insert_minutes: 20
+  update_minutes: 20
+  delete_minutes: 20
```

Comment:
Just a quick comment on this, do you happen to know how long does it to create this resource?

Reply:
it should only take a few seconds but with regional propagation, i think we say our SLOs are 7 minutes.

Reply:
Sounds good.